presto-docs/src/main/sphinx/connector/iceberg.rst

```diff
@@ -304,6 +304,76 @@ Property Name                                          Description
                                                        this size will not be cached.
 ====================================================   =============================================================   ============
 
+Alluxio Data Cache
+^^^^^^^^^^^^^^^^^^
+
+A Presto worker caches remote storage data in its original form (compressed and possibly encrypted) on local SSD upon read.
+
+The following configuration properties are required to set in iceberg catalog file (catalog/iceberg.properties):
+
+.. code-block:: none
+
+    cache.enabled=true
+    cache.base-directory=file:///mnt/flash/data
+    cache.type=ALLUXIO
+    cache.alluxio.max-cache-size=1600GB
+    hive.node-selection-strategy=SOFT_AFFINITY
+
+Jmx queries to get the metrics and verify the cache usage::
+
+    SELECT * FROM jmx.current."com.facebook.alluxio:name=client.cachehitrate,type=gauges";
+
+    SELECT * FROM jmx.current."com.facebook.alluxio:name=client.cachebytesreadcache,type=meters";
+
+    SHOW TABLES FROM jmx.current like '%alluxio%';
+
+File And Stripe Footer Cache
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Caches open file descriptors and stripe/file footer information in leaf worker memory. These pieces of data are mostly frequently accessed when reading files.
+
+The following configuration properties are required to set in iceberg catalog file (catalog/iceberg.properties):
+
+.. code-block:: none
+
+    # scheduling
+    hive.node-selection-strategy=SOFT_AFFINITY
+
+    # orc
+    iceberg.orc.file-tail-cache-enabled=true
+    iceberg.orc.file-tail-cache-size=100MB
+    iceberg.orc.file-tail-cache-ttl-since-last-access=6h
+    iceberg.orc.stripe-metadata-cache-enabled=true
+    iceberg.orc.stripe-footer-cache-size=100MB
+    iceberg.orc.stripe-footer-cache-ttl-since-last-access=6h
+    iceberg.orc.stripe-stream-cache-size=300MB
+    iceberg.orc.stripe-stream-cache-ttl-since-last-access=6h
+
+    # parquet
+    iceberg.parquet.metadata-cache-enabled=true
+    iceberg.parquet.metadata-cache-size=100MB
+    iceberg.parquet.metadata-cache-ttl-since-last-access=6h
+
+Jmx queries to get the metrics and verify the cache usage::
+
+    SELECT * FROM jmx.current."com.facebook.presto.hive:name=iceberg_parquetmetadata,type=cachestatsmbean";
+
+Metastore Versioned Cache
+^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Metastore cache would only work for Schema and Table Names since that is the information being fetched from Metastore. Other metadata would be fetched from the filesystem,
+which is present for hive caching will not take effect for those.
```

Comment:
I am afraid that I do not understand what is meant by ", which is present for hive caching will not take effect for those." Could you explain this a different way? 

As an alternative, perhaps this is not needed at all. Perhaps "Other metadata would be fetched from the filesystem." is enough?

Reply:
Yes, I think `Metastore cache only caches schema and table names. Other metadata would be fetched from the filesystem.` this clarifies that other metadata would not be cached and its limitation for this connector since other metadata is being fetched from the filesystem.

Thanks for your suggestions. I will make the changes accordingly.